ThunkLibs/GuestLibs/CMakeLists.txt

```
@@ -87,12 +97,22 @@ function(add_guest_lib NAME SONAME)
 
   ## Make signed overflow well defined 2's complement overflow
   target_compile_options(${NAME}-guest PRIVATE -fwrapv)
+  set (BITNESS_FLAGS "")
```

Nit: Newline here for separation from the `-fwrapv` setup.

```suggestion

  set (BITNESS_FLAGS "")
```

---

Oops, this was actually a remaining variable that was unused. Removed.